docs/install_guides/installing-openvino-pip.md

```diff
@@ -3,13 +3,14 @@
 You can install both OpenVINO™ Runtime and OpenVINO Development Tools through the PyPI repository. This page provides the main steps for installing OpenVINO Runtime.
 
 > **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
+
 ## Installing OpenVINO Runtime
 
-### System requirements
+### System Requirements
 
-Below you will find all supported operating systems and required Python\* versions. 
+Below you can find all the supported operating systems and the required Python versions. 
 
-|Operating Systems (64 bit)                        | [Python\* Version](https://www.python.org/downloads/) (64 bit)                 |
+|Operating System (64-bit)                        | [Python Version](https://www.python.org/downloads/) (64-bit)                 |
 |--------------------------------------------------|--------------------------------------------------------------------------------|
 |CentOS* 7                                         | 3.6, 3.7, 3.8                                                                  |
```

Comment:
I thought CentOS 7 is not supported, could you double-check? Also please help remove the asterisks (*) for brand/product names, as we are allowed to skip them in our doc site.

Reply:
Can't install it on CentOS 7. I've updated the documentation page.